Makefile

```diff
@@ -1,14 +1,14 @@
 SHELL = '/bin/bash'
-export BUILD_TAG ?= 1.2.1335-r3.5.1-python3.7.1-conda-10
+export BUILD_TAG ?= local
```

Comment:
nice, it makes sense as for example I've been changing this and the `docker-compose.yml` file locally ☺️ 

Speaking of `docker-compose.yml`, do you reckon the tag should match this as well there?

Reply:
The docker-compose.yml tag will inherit from here, so I think we're good in this case

Reply:
Oh nice, [I didn't see this](https://github.com/ministryofjustice/analytics-platform-rstudio/blob/main/docker-compose.yml#L6), I assumed it was same as in the `jupyter-lab` image ❤️